grafana/dashboards/prometheus-benchmark.json

```diff
@@ -0,0 +1,2755 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "Metrics useful for benchmarking and loadtesting Prometheus itself. Designed primarily for Prometheus 2.0.x",
+  "editable": true,
+  "gnetId": 3834,
+  "graphTooltip": 0,
+  "id": null,
+  "iteration": 1527022631341,
+  "links": [],
+  "panels": [
+    {
+      "aliasColors": {
+        "Chunks": "#1F78C1",
+        "Chunks to persist": "#508642",
+        "Max chunks": "#052B51",
+        "Max to persist": "#3F6833",
+        "Time series": "#70dbed"
+      },
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "${DS_PROMETHEUS}",
```

Comment:
I think that throughout this file you'll need to use:

```
      "datasource": "prometheus",
```

Otherwise I see this when I navigate to this dashboard:

![screen shot 2018-05-22 at 7 10 47 pm](https://user-images.githubusercontent.com/9226/40399873-7796fd3a-5df4-11e8-863e-f52408e8872e.png)

Reply:
nice catch thanks!